src/TeamSyncer.ts

```diff
@@ -135,13 +136,19 @@ export class TeamSyncer {
         await queue.addAll(syncFunctionPromises);
     }
 
-    private getTeamSyncConfig(teamId: string, item?: "channel"|"user", itemId?: string) {
+    private getTeamSyncConfig(teamId: string, item?: "channel"|"user", itemId?: string, isPrivate = false) {
         const teamConfig = this.teamConfigs[teamId] || this.teamConfigs.all;
         if (!teamConfig) {
             return false;
         }
-        if (item === "channel" && (!teamConfig.channels || !teamConfig.channels.enabled)) {
-            return false;
+        if (item === "channel") {
+            if (!teamConfig.channels?.enabled) {
+                return false;
+            }
+            if (teamConfig.channels.allow_private === false) {
+                // Default to true.
```

Comment:
What's this "Default to true" comment about?
If the condition is true, `false` is returned.

Reply:
Ah, because of `=== false` where `undefined` would lead to the condition being false.